index.bs

```diff
@@ -201,46 +201,52 @@ spec: webidl
 
   <section>
     <h3 id="reading-current-states">Reading the current permission state</h3>
-    <p>
-      |descriptor|'s <dfn export local-lt="state">permission state</dfn> is one
-      of {{"granted"}}, {{"prompt"}}, or {{"denied"}}, indicating respectively
-      if the calling algorithm should:
-    </p>
+    <div algorithm="read-current-permission-state">
+      <p>
+        To <dfn export>read the current permission state</dfn>, use the
```

Comment:
Can we use something like `|descriptor|'s <dfn export local-lt="state">permission state</dfn> is the result of the following algorithm.`? We *could* define two phrases that mean the same thing, but it seems redundant. I defined "permission state" as the name because @annevk wanted a really concise way to get to this value, in #74 request 2.